amqp/handlers.go

```diff
@@ -31,30 +31,10 @@ func UpdateUserHandler(del amqp.Delivery, dedb, icat *sqlx.DB, configuration *co
 	ctx, cancel := context.WithTimeout(context.Background(), time.Minute)
 	defer cancel()
 
-	dbs, rb, commit, err := db.NewBothTx(ctx, dedb, configuration.DBSchema, icat, configuration.UserSuffix, configuration.Zone, configuration.RootResourceNames)
-	if err != nil {
-		e := errors.Wrap(err, "Failed setting up database")
-		log.Error(e)
-		rejectErr := del.Reject(!del.Redelivered)
-		if rejectErr != nil {
-			log.Error(errors.Wrap(rejectErr, "Failed rejecting failed message"))
-		}
-		return e
-	}
-	defer rb()
```

Comment:
I'd love a review check that I'm still rolling back and committing transactions appropriately, since that code's all moved into the db/coordination.go file